src/MultiFormGoals/resources/js/components/multi-select-control/index.js

```diff
@@ -65,12 +67,14 @@ MultiSelectControl.propTypes = {
 	hideLabelFromVision: PropTypes.bool,
 	isLoading: PropTypes.bool,
 	isDisabled: PropTypes.bool,
+	placeholder: PropTypes.string,
 };
 
 MultiSelectControl.defaultProps = {
 	label: null,
 	value: null,
 	onChange: null,
+	placeholder: __( 'Select...', 'give' ),
```

Comment:
@henryholtgeerts minor detail - the `...` should not be a part of the translated string, but should be concatenated.

Reply:
Done, ref: https://github.com/impress-org/givewp/pull/5334/commits/566d6a7a46fe65a3553fe6873f6eb53ecfe47cfd